www/content/workstation/chef_vault.md

```diff
@@ -14,38 +14,38 @@ aliases = ["/chef_vault.html", "/chef_vault/"]
 
 [\[edit on GitHub\]](https://github.com/chef/chef-workstation/blob/master/www/content/workstation/chef_vault.md)
 
-`chef-vault` is a Ruby Gem that is included in Chef Workstation and Chef
-Infra Client. `chef-vault` allows the encryption of a data bag item by
+Chef Vault is a Ruby Gem that is included in Chef Workstation and Chef
+Infra Client. Chef Vault allows the encryption of a data bag item by
 using the public keys of a list of nodes, allowing only those nodes to
-decrypt the encrypted values. `chef-vault` uses the `knife vault`
+decrypt the encrypted values. Chef Vault uses the `knife vault`
 subcommand.
 
 {{< note >}}
 
-`chef-vault` does not currently support alternate keying mechanisms like
+Chef Vault does not currently support alternate keying mechanisms like
 GPG and Amazon KMS.
 
 {{< /note >}}
 
--   For more information about using the `chef-vault` cookbook, its
+-   For more information about using the chef-vault cookbook, its
     helper methods and resources, see
     <https://github.com/chef-cookbooks/chef-vault>
 
-The `chef-vault cookbook` is maintained by Chef Software. Use it along
-with `chef-vault` itself. This cookbook adds the `chef_vault_item`
+The chef-vault cookbook is maintained by Chef Software. Use it along
+with Chef Vault itself. This cookbook adds the `chef_vault_item`
 helper method to the Recipe DSL and the `chef_vault_secret` resource.
 Use them both in recipes to work with data bag secrets.
 
 {{< warning >}}
 
-Chef vault requires the use of Chef Infra Client configured to use
-public/private key pairs. Chef vault is incompatible with the practice
+Chef Vault requires the use of Chef Infra Client configured to use
+public/private key pairs. Chef Vault is incompatible with the practice
 of using Chef Infra Client with a private key as `client.pem` and a
 certificate set as its public identity in the Chef Infra Server
 database. To update existing nodes to use `chef-vault`, first
 re-register your Chef Infra Client nodes with the Chef Infra Server,
 which will generate public/private key pairs, and then install Chef
-vault on each node. Chef vault will generate the following error if used
+Vault on each node. Chef Vault will generate the following error if used
```

Comment:
See lines 54-55

It might be worth rewording it because it's a bit unclear. Maybe:

```
If Chef Vault is used with a Chef Infra Client instance that has
a `client.pem` private key and a certificate set as its
public identity in the Chef Infra Server database, Chef Vault will
generate the following error:
```

Reply:
I made some text changes that should fix this section up a bit.